bagvalidate/activity_test.go

```diff
@@ -110,26 +100,29 @@ func TestActivity(t *testing.T) {
 			},
 			want: bagvalidate.Result{
 				Valid: false,
-				Error: "invalid: Payload-Oxum validation failed. Expected 2 files and 38 bytes but found 1 files and 19 bytes",
+				Error: "invalid: payload-oxum validation failed. expected 2 files and 38 bytes but found 1 files and 19 bytes",
 			},
 		},
 	} {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			validator := setup(t)
+			validator := bagvalidate.NewValidator()
+
+			// Execute activity with test data.
 			ts := &temporalsdk_testsuite.WorkflowTestSuite{}
 			env := ts.NewTestActivityEnvironment()
 			env.RegisterActivityWithOptions(
 				bagvalidate.New(validator).Execute,
 				temporalsdk_activity.RegisterOptions{Name: bagvalidate.Name},
 			)
 
-			enc, err := env.ExecuteActivity(bagvalidate.Name, tt.params)
-			assert.NilError(t, err)
+			enc, _ := env.ExecuteActivity(bagvalidate.Name, tt.params)
```

Comment:
I think you should still `assert.NilError(t, err)` here.  If you don't catch an error, then `enc.Get(&result)` will panic below, and it can make it harder to debug the test.